addons/base_import/static/src/import_action/import_action.xml

```diff
@@ -29,7 +29,10 @@
                         importTemplates="importTemplates"
                         isBatched="isBatched"
                         onOptionChanged.bind="onOptionChanged"
+                        onOptionWebChanged.bind="onOptionWebChanged"
```

Comment:
```suggestion
                        onAdditionalFilesChanged.bind="onAdditionalFilesChanged"
```
? not sure to understand the "optionWeb" terminology

Reply:
* option is for the Python code/model (and so are forwarded in the request context)
* webOption are options only used by the WebClient side